### PR TITLE
Added 404 page

### PR DESCRIPTION
**Pull request by smyzin**
- added 404 page

https://user-images.githubusercontent.com/5114069/194376763-13339bbe-a2af-4624-afc5-181a894a4fed.mov

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -59,6 +60,12 @@ export default function () {
               <Route path="/wallet">
                 <BurnerWallet />
               </Route>
+              <Route path="/404">
```
Comment (T-Damer) [inline src/App.tsx:63]: Let's do it like this:
image

Comment (T-Damer) [reply src/App.tsx:63]: I mean multiple `path` as one prop

File: src/icons/OctoCorp.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+)
+const bluePath = classnames(stroke('stroke-primary'), strokeWidth('stroke-1.5'))
+
+export default function () {
+  return (
+    <svg
+      width="58"
+      height="58"
+      viewBox="0 0 58 58"
+      fill="none"
```
Comment (T-Damer) [inline src/icons/OctoCorp.tsx:20]: No need fot it

File: src/icons/OctoCorp.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      xmlns="http://www.w3.org/2000/svg"
+      className={dropShadow('drop-shadow-secondary')}
+    >
+      <path
+        d="M25.2308 1C11.5493 2.84919 1 14.6239 1 28.8737C1 44.4074 13.536 57 29 57C44.464 57 57 44.4074 57 28.8737C57 14.6239 46.4507 2.84919 32.7692 1"
+        className={commonPath}
+        stroke-linecap="round"
```
Comment (T-Damer) [inline src/icons/OctoCorp.tsx:27]: Let's remove:
```
        stroke-linecap="round"
        stroke-linejoin="round"
```

File: src/index.css [hunk 4]
```diff
@@ -189,6 +189,10 @@ circle {
   word-break: break-word;
 }
 
+.stroke-text-secondary {
```
[the remark this answers is not in the document]
Comment (T-Damer) [reply src/index.css:192]: Btw, we can use svg here, it's pretty small

File: src/App.tsx (hunk 1 above)
Comment (1Damer) [reply src/App.tsx:63]: Meh, deprecated

File: src/index.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (T-Damer) [reply src/index.css:207]: I used svg, because it's faster than building bg gradients from scratch

**Pull request merged**